src/pages/experiments/[experimentId]/data-processing/index.jsx

```diff
@@ -475,6 +475,7 @@ const DataProcessingPage = ({ experimentId, experimentData, route }) => {
                       steps[stepIdx].key,
                       { enabled: !processingConfig[steps[stepIdx].key]?.enabled },
                     ));
+                    dispatch(saveProcessingSettings(experimentId, steps[stepIdx].key));
```

Comment:
If we made some changes to the filter settings, clicked disable, clicked enable and then ran the pipeline wouldn't we be pre-saving the filter settings (since `saveProcessingSettings` sends everything to the api)?

Reply:
Correct, however there are many other inconsistencies in the way we do it. @vickymorrison is working on a new UX to fix this, see this discussion:

https://this-is-biomage.slack.com/archives/C015CHSUDRU/p1619625304189300

and the mockups:

https://docs.google.com/presentation/d/1EIBqwhBJRbnOPumUixbhzAx6LgCSlFkjuhatMB6CveQ

Reply:
That is to say, we will make it so that the button will not be displaying "save" but rather "run" all the time.

Reply:
Ok, and I guess it doesn't really matter that much what we save to processing settings since the filter is disabled anyways.